pkg/console/operator/sync_v400.go

```diff
@@ -101,10 +101,10 @@ func (co *consoleOperator) sync_v400(ctx context.Context, controllerContext fact
 		return statusHandler.FlushAndReturn(customLogoError)
 	}
 
-	defaultIngressCertConfigMap, defaultIngressCertErrReason, defaultIngressCertErr := co.ValidateDefaultIngressCertConfigMap(ctx)
-	statusHandler.AddConditions(status.HandleProgressingOrDegraded("DefaultIngressCertValidation", defaultIngressCertErrReason, defaultIngressCertErr))
-	if defaultIngressCertErr != nil {
-		return statusHandler.FlushAndReturn(defaultIngressCertErr)
+	oauthServingCertConfigMap, oauthServingCertErrReason, oauthServingCertErr := co.ValidateOAuthServingCertConfigMap(ctx)
+	statusHandler.AddConditions(status.HandleProgressingOrDegraded("OAuthServingCertValidation", oauthServingCertErrReason, oauthServingCertErr))
```

Comment:
you're changing a condition name here, you'll want to use something like https://github.com/openshift/library-go/blob/14aa5b749d8ddbae509ea97c988228934dbbd3db/pkg/operator/staleconditions/remove_stale_conditions.go#L19 in order to remove a stale condition (otherwise this might brick upgrades)

Reply:
@stlaz good catch. I've added the old condition to our stale condition controller here: https://github.com/openshift/console-operator/pull/571/files#diff-509903f93e97151f80c21647a3701cfd7db4dc3f6723a6d3a3d897a3b32a7832R352